### ISSUE TITLE
DRUPAL: About Disability Ratings/ Effective Date page- Structure Change Needs to be fixed in Drupal

### ISSUE THREAD

**Issue by Karak888** [department-of-veterans-affairs/va.gov-team#2449]
### **Background:**
It was discovered that there was a change to the structure of the Effective Date page on the About Disability ratings (/disability/about-disability-ratings/effective-date/ changes to /disability/effective-date/), which has moved the page out from being a child of the About Disability Ratings page, and also affects the left nav.  Being that this page is in the CMS (along with the rest of the disability hub), this will need to be handled entirely from Drupal. 

### **_Important Note:_** 
Will need to implement a redirect at the time we change the URL so that we do not lose users to a 404 error.

**Comment by Karak888** [comment]
Will keep this in backlog to discuss during grooming. Thanks @mnorthuis thanks @ncksllvn 

:cc @jenniferlee-dsva

**Comment by mnorthuis** [comment]
This page: `/disability/about-disability-ratings/effective-date/`

Should be here: `/disability/effective-date/`

**Comment by Karak888** [comment]
doh! Thanks @mnorthuis !!

**Comment by RLHecht** [comment]
@mnorthuis For the left nav of this page, should it now reside under More Resources and in what order in there should it go?

**Comment by mnorthuis** [comment]
@RLHecht Yes, it should go under More Resources, and it should be the second option, so from the screen shot you sent me, just move it up one spot so it is after the "About disability ratings" info.  That puts the general "about" type content grouped together, then the actual rates, then other resources. 

![image](https://user-images.githubusercontent.com/20994159/67024399-f91c8780-f0c9-11e9-83fd-21e5cf112a77.png)

**Comment by RLHecht** [comment]
Thanks @mnorthuis. This is now moved. Here's the screenshot: 
<img width="316" alt="Screen Shot 2019-10-17 at 11 37 22 AM" src="https://user-images.githubusercontent.com/48728214/67025147-87e1d200-f0d3-11e9-807a-06ecdfc980dd.png">

I'll move this to validate and will check once it's live on the site. @Karak888

**Comment by mnorthuis** [comment]
@RLHecht @Karak888 Is the redirect from the old URL to the new URL being implemented with this change?  Or does this just change its position in the left nav?

**Comment by RLHecht** [comment]
@mnorthuis @Karak888 The new URL should take the place of the old one but there was a bug that occurred on Thursday after I moved it. I'll check with the CMS team again today to see if it's fixed. I just checked the live site and the URL is not showing up at the moment. I'll let you both know once it's fixed and working.

**Comment by mnorthuis** [comment]
@RLHecht My understanding is that drupal does not currently handle redirects.  The change in drupal will change the URL of the page, but we need to redirect the old URL to the new URL to avoid 404s.

**Comment by Karak888** [comment]
@RLHecht do you need help with the redirect work?  Thanks!!

**Comment by Karak888** [comment]
@mikki This isn't an isolated case here right? If the case is that drupal does not currently handle redirects. Per your note above:  The change in drupal will change the URL of the page, but we need to redirect the old URL to the new URL to avoid 404s.  If this is not specific to this page maybe we can track these issues under a new Epic.  Please let me know your thoughts! thanks

:cc @jenniferlee-dsva @RLHecht

**Comment by mnorthuis** [comment]
This is not specific to this page.  Drupal does not do redirects at this time.   This redirect needs to be handled just like all the others as we get that approach finalized.  

This means that this change cannot go live until we are ready to do the redirect in parallel with it, or users risk the chance of encountering a 404 if they use an old link or bookmark.

**Comment by RLHecht** [comment]
@mnorthuis @Karak888 @jenniferlee-dsva This is now working correctly with the page showing up in the proper place on the left nav menu. We still need to implement the redirect work for this ticket.

**Comment by RLHecht** [comment]
@mnorthuis @meganhkelley The rest of this ticket is done and now needs the redirect work done.

**Comment by mnorthuis** [comment]
Redirect handled in issue https://github.com/department-of-veterans-affairs/va.gov-team/issues/3805

**Comment by RLHecht** [comment]
@Karak888 with the redirect being handled on another ticket, i think we can close this one. It's validated on the live site.

**Comment by Karak888** [comment]
thanks @RLHecht and @mnorthuis closing this now..
